src/utils/logging.c

```diff
@@ -75,3 +78,34 @@ void bd_utils_log_format (gint level, const gchar *format, ...) {
 
     g_free (msg);
 }
+
+/**
+ * bd_utils_log_stdout:
+ * @level: log level
+ * @msg: log message
+ *
+ * Covenient function for logging to stdout. Can be used as #BDUtilsLogFunc.
+ *
+ */
+void bd_utils_log_stdout (gint level, const gchar *msg) {
+    switch (level) {
+        case BD_UTILS_LOG_DEBUG:
+#ifdef DEBUG
+            g_debug ("%s", msg);
+#endif
+            break;
+        case BD_UTILS_LOG_INFO:
+        case BD_UTILS_LOG_NOTICE:
+            g_info ("%s", msg);
+            break;
```

Comment:
This will produce a lot of noise from running external utilities and doing DBus method calls, right? It should also be guarded with the `#ifdef DEBUG` I think.

Reply:
I think the "stdout" logging should be the same as the default logging. If we want to avoid stdout spamming we should move the exec logs to debug (it actually is more debug than info IMHO) and maybe change the debug logging to runtime instead of build time configurable.
Or we could add new option to set default log level and discard everything below it (e.g. set default to warning and let user choose if they want to log info/debug etc.). Both cryptsetup and libdevmapper have similar functionality.

Reply:
> If we want to avoid stdout spamming we should move the exec logs to debug (it actually is more debug than info IMHO)

I think this goes back to Blivet/Anaconda where stdout from spawned processes is `INFO`, IIRC. I agree `DEBUG` would make more sense.

Reply:
> change the debug logging to runtime instead of build time configurable.
> Or we could add new option to set default log level and discard everything below it (e.g. set default to warning and let user choose if they want to log info/debug etc.).

Sounds like a good idea! :+1: